src/django_dramatiq_pg/models.py

```diff
@@ -1,31 +1,24 @@
-from enum import Enum
-
 from django.contrib.postgres.fields import JSONField
 from django.db import models
 from django.utils import timezone
 
 
-class State(str, Enum):
-    QUEUED = "queued"
-    CONSUMED = "consumed"
-    REJECTED = "rejected"
-    DONE = "done"
-
-
-State.choices = tuple((state.value, state.name) for state in State)
+class BackgroundJob(models.Model):
 
+    class STATUS(models.TextChoices):
```

Comment:
I'd recommend calling this STATE. No point fighting dramatiq's nomenclature.
We can handle that with mapping in our own code.